SocialFilter/sections/specific.txt

```diff
@@ -63,6 +63,8 @@ kakoy-smysl.ru,tutsvarka.ru,tuttotech.net,allets.ru,annahelp.ru,newslenta.com,ko
 !
+lovepik.com##.ContentShare
```

Comment:
Could you share a screenshot with this element? I don't see it.

Reply:
https://kr.lovepik.com/image-400488286/girl-looking-up-at-the-stars.html

![Screenshot from 2023-01-30 10-27-20](https://user-images.githubusercontent.com/98787049/215452015-ffd546cf-10df-47b8-b221-5d26d42d21da.png)